web/pgadmin/tools/psql/__init__.py

```diff
@@ -394,6 +394,18 @@ def get_conn_str_win(manager, db):
     manager.export_password_env('PGPASSWORD')
     db = db.replace('"', '\\"')
     db = db.replace("'", "\\'")
+
+    hostaddr = manager.get_connection_param_value('hostaddr')
```

Comment:
`get_connection_param_value` can be modified to return connection param dict if param name is not passed.
For eg - `manager.get_connection_param_value()` will return all the conn params. And then the returned dict can be used to get params. This way, multiple `get_connection_param_value` calls can be avoided.

Reply:
For this specific file, I have removed the function completely so no need to fix the above as in the other places only one function call is there. I am keeping this as it is.